### ISSUE TITLE
[Usage] LLaVA-v1.6-vicuna-7b generates incomplete sentence

### ISSUE THREAD

**Issue by gymbeijing** [haotian-liu/LLaVA#1651]
### Describe the issue

Hi Haotian, thank you for the great work!

I've been trying to use LLaVA to generate rationale for my multimodal inputs (a news image + a news caption).

The prompt looks like this:
```
Given the news Image and the news Text ([T]), please provide a streamlined explanation associated with the text and the image by using the contextual background commonsense knowledge, to explicitly explain how the news image is reasoned as out-of-context.
```

Here is an example. In the following example, ```[T]``` is
```
Passengers are stranded outside the Guangzhou railway station in China s Guangdong province on Tuesday
```

Given the prompt and the news image, the model's output is 
```
 .
2. The image could be a stock photo or a generic image of a crowd, which is not specific to the news event.
3. The image might be used to illustrate a broader concept or story, but without additional context, it does not directly relate to the news text.

In summary, the image does not provide any specific information about the passengers stranded outside the Guangzhou railway station, and thus, it is out of context with the news text.
```

Clearly, the output starts from the middle of a sentence. This also happens with different multimodal news inputs (which I didn't illustrate here).

Some hyperparameters: ```--sep , --temperature 0 --max_new_tokens 512```

Could you please advise me where I can adjust the parameter or my generation code (if needed I can share)? Thank you in advance.

**Comment by gymbeijing** [comment]
This is due to a bug in my code. I truncated the output sentence from the middle.
